.github/workflows/wheel.yml

```diff
@@ -0,0 +1,79 @@
+name: old_wheel_creation
+
+on:
+  workflow_dispatch:
+
+env:
+  PACKAGE_NAME: maliput_py
+  ROS_DISTRO: foxy
+  ROS_WS: maliput_ws
+
+jobs:
+  wheel_generation:
+    name: Wheel generation
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu:20.04
+    steps:
+    # setup-ros first since it installs git, which is needed to fetch all branches from actions/checkout
+    - uses: ros-tooling/setup-ros@v0.6
+    - uses: actions/checkout@v3
+      with:
+        path: ${{ env.ROS_WS }}/src/${{ env.PACKAGE_NAME }}
+    # clone public dependencies
+    - name: vcs import
+      shell: bash
+      working-directory: ${{ env.ROS_WS }}
+      run: vcs import src < src/${PACKAGE_NAME}/.github/dependencies.repos
+    - name: check if dependencies have a matching branch
+      shell: bash
+      working-directory: ${{ env.ROS_WS }}/src
+      run: ./${PACKAGE_NAME}/.github/try_vcs_checkout ${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}} .
+    - run: colcon graph
+      shell: bash
+      working-directory: ${{ env.ROS_WS }}
+    - name: rosdep install
+      shell: bash
+      working-directory: ${{ env.ROS_WS }}
+      run: |
+        rosdep update --include-eol-distros;
+        rosdep install  -i -y --rosdistro ${ROS_DISTRO}  --from-paths src
+    - name: colcon build up-to
+      shell: bash
+      working-directory: ${{ env.ROS_WS }}
+      run: |
+        . /opt/ros/${ROS_DISTRO}/setup.bash;
+        colcon build --packages-up-to ${PACKAGE_NAME} \
+          --packages-skip ${PACKAGE_NAME} \
+          --event-handlers=console_direct+
+    - name: install extra dependencies
```

Comment:
Up to here, can't we just upload the artifact of the normal build.yml and then continue from there in this workflow?

Reply:
Done [6924103](https://github.com/maliput/maliput_py/pull/82/commits/69241036a5b9fae3f2b779f92f7c06b1de6f4dfc)